### PR TITLE
IRIS-4356 | Dump follows for threads

### PR DESCRIPTION
**Pull request by slayful**

### CODE REVIEW

File: extensions/wikia/Discussions/maintenance/ForumDumper.php [hunk 1]
```diff
@@ -311,19 +318,44 @@ public function getVotes() {
...
 
 		return $this->votes;
 	}
+
+	public function getFollows() {
+		$this->threadIds = [];
+
+		foreach ($this->getPages() as $page) {
+			if ($page->namespace === NS_WIKIA_FORUM_BOARD_THREAD) {
+				$threadIds[$page->raw_title] = $page->id;
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/ForumDumper.php]: MAJOR Remove this unused "$threadIds" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

File: extensions/wikia/Discussions/maintenance/ForumDumper.php [hunk 2]
```diff
@@ -9,7 +9,9 @@ class ForumDumper {
 	const TABLE_REVISION = 'revision';
 	const TABLE_TEXT = 'text';
 	const TABLE_VOTE = 'page_vote';
-	const TABLE_PAGE_WIKIA_PROPS  = 'page_wikia_props';
+	const TABLE_PAGE_WIKIA_PROPS = 'page_wikia_props';
+	//TODO verify that this is the table
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/ForumDumper.php]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Review by jsutterfield — APPROVED** [review]

**Comment by jsutterfield** [comment]
Looks great!

File: extensions/wikia/Discussions/maintenance/ForumDumper.php [hunk 3]
```diff
@@ -76,12 +76,19 @@ class ForumDumper {
...
+	const COLUMNS_FOLLOWS = [
+		"follower_id",
+		"mw_thread_id",
+		"timestamp",
 	];
 
 	private $pages = [];
 	private $revisions = [];
 	private $votes = [];
+	private $threadIds = [];
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/ForumDumper.php]: MAJOR Remove this unused "$threadIds" private field. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**